### PR TITLE
add new sql question(s)

### PR DESCRIPTION
**Pull request by Tryxns**
## Checklist

- [x] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [x] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [x] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [x] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [x] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/sql-quiz.ts [hunk 1]
```diff
@@ -1201,7 +1201,27 @@ const sqlQuiz = [
     Explanation:
       "Temporary tables are automatically removed from the database system at the end of the session, preventing any manual cleanup and ensuring efficient use of database resources",
     Link: "https://www.freecodecamp.org/news/sql-temp-table-how-to-create-a-temporary-sql-table/"
-  }
+  },
+  {
+    Question: "In MySql, which command to show all the default databases that come as part of the installation?",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:1206]:
```suggestion
    Question: "In MySQL, which command is used to show all of the default databases that come as part of the installation?",
```

File: src/data/sql-quiz.ts [hunk 2]
```diff
@@ -1201,7 +1201,27 @@ const sqlQuiz = [
...
+    Answer: "show databases",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:1207]:
```suggestion
    Answer: "SHOW DATABASES",
```

File: src/data/sql-quiz.ts [hunk 3]
```diff
@@ -1201,7 +1201,27 @@ const sqlQuiz = [
...
+    Distractor1: "databases",
+    Distractor2: "view databases",
+    Distractor3: "show database",
+    Explanation:
+      "SHOW DATABASES command which shows all the default databases that come as part of the installation",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:1212]:
```suggestion
      "The SHOW DATABASES command shows all the default databases that comes with the MySQL installation.",
```

File: src/data/sql-quiz.ts [hunk 4]
```diff
@@ -1201,7 +1201,27 @@ const sqlQuiz = [
...
+    Link: "https://youtu.be/ER8oKX5myE0?si=gfwnDC7sCi6p5YIc&t=860"
+  },
+  {
+    Question: "In MySql, which command to shows the current values of MySQL system variables?",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:1216]:
```suggestion
    Question: "In MySQL, which command is used to show the current values of MySQL system variables?",
```

File: src/data/sql-quiz.ts [hunk 5]
```diff
@@ -1201,7 +1201,27 @@ const sqlQuiz = [
...
+    Answer: "show variables",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:1217]:
```suggestion
    Answer: "SHOW VARIABLES",
```

File: src/data/sql-quiz.ts [hunk 6]
```diff
@@ -1201,7 +1201,27 @@ const sqlQuiz = [
...
+    Distractor1: "vars",
+    Distractor2: "variables",
+    Distractor3: "show vars",
+    Explanation:
+      "show variables is the command to check the current value of MySQL system variables",
```
Comment (jdwilkin4) [inline src/data/sql-quiz.ts:1222]:
```suggestion
      "The SHOW VARIABLES statement is used to check the current value of MySQL system variables.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Tryxns** [comment]
HI @jdwilkin4 , thanks for review. I have resolved all change requests based on your suggestions, please re-review

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**